src/OpenColorIO/ops/fixedfunction/FixedFunctionOpGPU.cpp

```diff
@@ -192,6 +192,179 @@ void Add_Glow_03_Inv_Shader(GpuShaderText & ss, float glowGain, float glowMid)
     ss.newLine() << "outColor.rgb = outColor.rgb * glowGainOut + outColor.rgb;";
 }
 
+void Add_GamutMap_13_Compress(GpuShaderText & ss,
+                              const char * dist,
+                              const char * comprDist,
+                              float lim,
+                              float thr,
+                              float power,
+                              float invert)
+{
+    ss.newLine() << "{";
+    ss.indent();
+
+    ss.newLine() << "float lim = " << lim << ";";
+    ss.newLine() << "float thr = " << thr << ";";
+    ss.newLine() << "float pwr = " << power << ";";
```

Comment:
Are these three local variables needed?  The method argument can be used directly in the code.

Reply:
I used that to ease the writing of the function but I agree it make no sense for the code generation.
Fixed.